egs/mgb2/ASR/local/compile_hlg.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
```

Comment:
Could you replace it with a symlink to the file from librispeech?

Reply:
Done